public/app/plugins/datasource/cloudwatch/aws_url.ts

```diff
@@ -11,8 +11,23 @@ export interface AwsUrl {
   source: string[];
 }
 
+const defaultURL = 'console.aws.amazon.com';
+const usGovURL = 'console.amazonaws-us-gov.com';
+const chinaURL = 'console.amazonaws.cn';
+
+export function getLogsEndpoint(region: string): string {
+  let url = defaultURL;
+  if (region.startsWith('us-gov-')) {
```

Comment:
it feels like a similar thing should exist in https://github.com/aws/aws-sdk-js so we wouldn't have to have hard coded strings here but Idk I did a quick look and didn't see it so idk maybe it doesn't! The closest I found was [this](https://github.com/aws/aws-sdk-js/blob/94e4ab32e4d1faac725968bbeff9329f68a9bc20/lib/region_config_data.json) but I'm not sure how that would be used. 

Anyway probably not worth it!